containers/placeholder/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx
```

Comment:
I copied this from [here](https://github.com/trisacrypto/testnet/blob/main/containers/placeholder/landing/Dockerfile).